scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -246,6 +251,44 @@ def create_swap(self, swap_dev):
     sys.stdout.write(f"Adding {swap_dev} swap device to {FSTAB}\n")
     add_swap_fstab(self, swap_dev)
 
+'''
+    Create swap disks of VG.
+    If swap size is more than 16T then
+    create more than one swap disks each of max 16T.
+'''
+def create_swap_disks(self, index, swap_sz, vg_name):
+    disk_count = 1
+    while(swap_sz > MAX_SWAP_DISK_SZ):
+        alloc_swap_sz = MAX_SWAP_DISK_SZ
+        swap_sz -= MAX_SWAP_DISK_SZ
+        lv_swap_name = f"lv_main_swap{index}_{disk_count}"
+        cmd = f"lvcreate -n {lv_swap_name} {vg_name} -L {alloc_swap_sz}T --yes"
+        execute_command(self, cmd)
+        disk_count += 1
+        swap_dev = f"/dev/{vg_name}/{lv_swap_name}"
+        create_swap_entries(self, swap_dev)
+    lv_swap_name = f"lv_main_swap{index}_{disk_count}"
+    swap_dev = f"/dev/{vg_name}/{lv_swap_name}"
+    swap_sz = round(swap_sz, 3)
+    cmd = f"lvcreate -n {lv_swap_name} {vg_name} -L {swap_sz}T --yes"
+    execute_command(self, cmd)
+    create_swap_entries(self, swap_dev)
```

Comment:
I know this is to handle the last one which is less than "MAX_SWAP_DISK_SZ".

But I think we can avoid duplicating the code.
Can we do this in the loop?